Yosemite/Yosemite/Tools/Media/MediaAssetExporter.swift

```diff
@@ -112,7 +113,7 @@ private extension MediaAssetExporter {
                 return nil
         }
         guard allowableFileExtensions.contains(fileExtensionForType) else {
-            return kUTTypeJPEG as String
+            return UTType.jpeg.identifier as String
```

Comment:
nit: it looks like `identifier` is already a `String`, maybe we don't need another type casting?

```suggestion
            return UTType.jpeg.identifier
```